ibm/data_source_ibm_is_lb.go

```diff
@@ -379,6 +386,7 @@ func lbGetByName(d *schema.ResourceData, meta interface{}, name string) error {
 		if *lb.Name == name {
 			d.SetId(*lb.ID)
 			d.Set(isLBName, *lb.Name)
+			d.Set(isLBLogging, *lb.Logging.Datapath.Active)
```

Comment:
Add a condition to check if lb.Logging.!= nil && lb.Logging.Datapath != nil  {d.Set(isLBLogging, *lb.Logging.Datapath.Active)} and then set the value